package.json

```diff
@@ -49,6 +49,6 @@
     "graphql": "15.5.3"
   },
   "peerDependencies": {
-    "ethers": "^5.4.6"
+    "ethers": "5.4.6"
```

Comment:
Keep the `^` here.

Reply:
This is a peer dependency, so `^` is crucial since it allows the consumer to use any version of ethers that's above `5.4.6` but below `5.5.x`